.github/workflows/flake8.yml

```diff
@@ -0,0 +1,26 @@
+name: flake8 
+
+on:
+  push:
+    paths:
+      - '*.py'
+
+jobs:
+  flake8_py3:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.9
+          architecture: x64
+      - name: Checkout PyTorch
+        uses: actions/checkout@master
+      - name: Install flake8
+        run: pip install flake8 flake8-bugbear flake8-pyproject
```

Comment:
Maybe `pip install -r ./REQUIREMENTS.qa.txt` here to de-duplicate the QA dependencies?

Reply:
In this case we need to checkout the repo first. I've added that step and now install the `REQUIREMENTS.qa.txt`